ubuntu-wsl2-systemd-script.sh

```diff
@@ -20,6 +20,7 @@ Defaults        env_keep += WSLENV
 Defaults        env_keep += WSL_INTEROP
 Defaults        env_keep += WSL_DISTRO_NAME
 Defaults        env_keep += PRE_NAMESPACE_PATH
+Defaults	env_keep += PRE_NAMESPACE_PWD
```

Comment:
```suggestion
Defaults        env_keep += PRE_NAMESPACE_PWD
```

Nit-picking: the spacing here is with space characters, not tabs.. not important to be fixed, but a nice to have.

Reply:
Sure, I noticed it myself. In the projects I normally work with I see tabs, and in my terminal they aligned as well. Will fix.

Reply:
Fixed.